MilitaryManager.Attachments.API/Controllers/AttachmentsController.cs

```diff
@@ -60,5 +69,14 @@ public string GenerateNewDocument()
 
             return $"https://{Request.Host}/api/attachments/find?name={docName}";
         }
+
+        [HttpPost]
```

Comment:
The path is incorrect. We cannot use this implementation in Angular. Change it to [Route("{id}")] or don't specify the path at all. To understand how we use controllers look at Angular base.services.ts lines 81-87, 106-144